assets/script.js

```diff
@@ -32,5 +45,8 @@ fetch("https://api.petfinder.com/v2/oauth2/token", {
   })
 })
 //api.petfinder.com/v2/{CATEGORY}/{ACTION}?{parameter_1}={value_1}&{parameter_2}={value_2}
+}
+
 
-var dogKey = "7VT9G3psGTVpzFOhgUZsag==6qGoaeaUyBn1jA8n"
+// searchFormEl.addEventListener('submit', function (event) { getWeather(event, searchInput.value) });
```

Comment:
Looks good except for getweather

Reply:
yeah that was a commented out copy of my notes I just saw